### ISSUE TITLE
Accessibility issues: missing alt text for images & screen reader issues in stash block

### ISSUE THREAD

**Issue by Nukker1** [FMCorz/moodle-block_stash#28]
Stash Block

1. Missing alternative text for images:
There is currently no option to provide alternative (alt) text for images of stash items. This makes it difficult or impossible for users relying on screen readers to understand the content or of these images.

2. Item names not read by screen reader in stash block (inventory):
Object names are only read by the screen reader as long as the stash items are in the content area. However, in the inventory of the stash block, while the object names are visible, they are not detected by the screen reader. As a result, users cannot interact with these items using only the screen reader unless they click on them directly.

Suggested Fixes:

1. For missing alt text for images:
Implement a feature that allows alt text to be added to images of Stash items, improving accessibility for users who rely on screen readers.

2. For item names in stash block inventory:
Ensure that object names in the stash block inventory are properly recognized by screen readers, allowing users to interact with inventory items via keyboard navigation.

**Comment by FMCorz** [comment]
@Nukker1 Thanks for reporting this. This repository is outdated, the development is currently taking place at [abgreeve/moodle-block_stash](https://github.com/abgreeve/moodle-block_stash/issues). May I suggest to report the issue there? Thank you!
